### DISCUSSION TITLE
Fallout 3 GOTY crashing

### DISCUSSION THREAD

**Discussion by Anastazbnko** [legendary-gl/legendary#488] [Q&A] [answered]
When I start a New Game it crashes. Don't have any saves so I don't know if it'll crash when I continue a game.
Does this happen to anyone else?

**Comment by CommandMC** [comment, marked as answer]
Depending on your GPU, you might need https://www.nexusmods.com/fallout3/mods/24913

**Reply by Anastazbnko** [reply]
It works, thank you legend
